_src/pattern-library/sass/utilities/_variables.scss

```diff
@@ -331,6 +332,46 @@ $buttons-elevated-shadow: 3px !default;
 $buttons-border-radius: 3px !default;
 $buttons-font-weight: font-weight(semi-bold) !default;
 
+// forms
+$label-color: $text-base-color;
```

Comment:
I think you forgot one more ``!default`` flag on this ``$label-color`` variable.